### PR TITLE
[WIP] 2019 webpage

### PR DESCRIPTION
**Pull request by casassg**
## What this PR does / Why we need it
Release 2019 version. This PR is just to easily oversee the changes done in the repo and allow for quicker review. 

## Which issue(s) this PR fixes (optional) 

None afaik

## Some questions
- [x] I have read the contributing guidelines
- [x] I abide by this repository Code of Conduct
- [x] I have checked all margins within the sponsor/partners area

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,28 +114,97 @@ Features included
...
 
 Clients will poll constantly the schedule.json file. To update the schedule just change the file and deploy.  
 New versions of the schedule will be loaded only if 'version' is different.
 
-# License
+## Tech stack
+
+This are the explanations of why we chose each technology, so you can decide whether keep it or change it.
+
+### Conventions
```
Comment (casassg) [inline README.md]: Maybe move conventions to https://github.com/hackupc/frontend/blob/master/.github/CONTRIBUTING.md

Comment (mauriciabad) [reply README.md]: Okey, I'll continue them there.

**Review by mauriciabad — APPROVED** [review]
OK

File: src/index.html [hunk 2]
```diff
@@ -311,7 +311,8 @@ <h2 class="section__title">Partners</h2>
   <script src="/assets/js/cookies.min.js" async></script>
 
   <!-- Easter Egg -->
-  <i class="easter-egg">🐝 BIENE HACKUPC AGAIN! 💃🏽 OH NO YOU SAW ME 😱. IF YOU ARE FROM UPC 🎓 PLS SEND AN EMAIL 📧 TO DEVS@HACKUPC.COM 💖</i>
+	<i class="easter-egg">🐝 BIENE HACKUPC AGAIN! 💃🏽 OH NO YOU SAW ME 😱. IF YOU ARE FROM UPC 🎓 PLS SEND AN EMAIL 📧 TO DEVS@HACKUPC.COM 💖</i>
+	<span class="easter-egg">Gerard was here. Wondering if anyone will see this before production. Nothing to see here....</span>
```
Comment (mauriciabad) [inline src/index.html]: Aha! Gotcha!

Comment (casassg) [reply src/index.html]: Cassssiiii

**Pull request merged**